### DISCUSSION TITLE
How to "unstack" photos

### DISCUSSION THREAD

**Discussion by ziggysid** [photoprism/photoprism#4635] [Q&A]
PhotoPrism has stacked some of my photos, and I don't like it. The important question is how can I separate them? New to PhotoPrism so wasn't really aware of this Stacking feature until it happened. After they got stacked I disabled "Place & Time", "Unique ID" and "Sequential name" stacking options. Then tried to rebuild the index, first I tried Library>Index>Complete Rescan, but the are still stacked. Next I tried full index rebuild from the command line. Still they are stacked.
Any suggestions?

**Comment by omerdduran** [comment]
Hi! Changing the stacking settings and reindexing won't automatically unstack existing stacks. So there are two options you can do instead. One is to manually remove the stacking feature on each image. ( Navigate to Search > Stacks in the main menu, then open the photo's edit dialog. In the Files tab, you can unstack ) The other is to do a complete reset with the `photoprism reset` command. Just remember that this will reset any manual tags, face recognition data, and personal settings you've added.

Also, please note that files with identical names will always remain stacked. This cannot be disabled as it's required for Live Photos and RAW+JPEG pairs to work properly.

https://docs.photoprism.app/getting-started/docker-compose/#command-line-interface
https://docs.photoprism.app/getting-started/docker-compose/#command-line-interface

**Reply by ziggysid** [reply]
Thanks. I didn't know that "unstack" button existed before. That's done the job. Fortunately I only had a handful of photos that were stacked, otherwise I could imagine it will be a laborious task.
